src/ic-websocket-gateway/src/gateway_metrics.rs

```diff
@@ -0,0 +1,61 @@
+use std::error::Error;
+use std::time::Duration;
+use metrics::{describe_gauge, describe_histogram, gauge};
+use metrics_exporter_prometheus::PrometheusBuilder;
+use metrics_util::{MetricKindMask};
+// use opentelemetry::{global, KeyValue};
+// use opentelemetry_otlp::{Protocol, WithExportConfig};
+
+pub fn init_metrics() -> Result<(), Box<dyn Error>> {
+    let builder = PrometheusBuilder::new();
+    builder
```

Comment:
Can we add the possibility to specify a different listening port via arguments for the prometheus endpoint?

We can keep the 9000 port as default in the arguments, so that it's not mandatory